components/map/mapbox-map.tsx

```
@@ -584,6 +589,45 @@ export const Mapbox: React.FC<{ position?: { latitude: number; longitude: number
 
 
 
+  // Handle note markers from MapDataContext
+  useEffect(() => {
+    if (!map.current || !initializedRef.current) return;
+
+    // Clear existing note markers
+    noteMarkersRef.current.forEach(marker => marker.remove());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)